vpr/src/route/route_tree.h

```diff
@@ -87,6 +87,7 @@
 #include <unordered_map>
 
 #include "connection_based_routing_fwd.h"
+#include "lookahead_profiler.h"
```

Comment:
can't this be replaced by a forward declaration?

Reply:
Forward declaration here creates compiler errors